composer/trainer/trainer.py

```diff
@@ -159,17 +400,29 @@ class Trainer:
                 desired optimization batch size is ``2048`` and training is happening across 8 GPUs, then each
                 ``train_dataloader`` should yield a batch of size ``2048 / 8 = 256``. If ``grad_accum = 2``,
                 then the per-rank batch will be divided into microbatches of size ``256 / 2 = 128``.
-        max_duration (int, str, or Time): The maximum duration to train. Can be an integer, which will be
+
+            If ``train_dataloader`` is not specified when constructing the trainer, it must be specified when invoking
+            :meth:`.Trainer.fit`.
+        train_dataloader_label (str, optional): The label for the train dataloader. (default: ``'train'``)
```

Comment:
```suggestion
        train_dataloader_label (str, optional): The label for the train dataloader. Default: ``'train'``.
```

Reply:
You're right -- we do need to unify how we document defaults -- not quite sure what the google style is (I think it's a full sentence `Defaults to XXX.`), but need to verify this. Going to leave the defaults as-is as we use this style in other parts of the codebase, but this is something we should address later.